mycoffee/functions.py

```diff
@@ -117,15 +121,16 @@ def check_ratio_limits(params):
 
     :param params: parameters
     :type params: dict
-    :return: None
+    :return: result as bool (True --> the ratio is out of range)
```

Comment:
Isn't it better to make `False --> the ratio is out of range`? It makes more sense for me.

Reply:
Good point 💯 
Fixed in https://github.com/sepandhaghighi/mycoffee/pull/29/commits/6cf1c8cd27495612b7c8b2184645d08652ffd0a6